docs/source/en/training/lora.md

```diff
@@ -378,6 +378,56 @@ images_fusion = pipe(
 ).images
 ```
 
+## Working with multiple LoRA checkpoints
+
+<Tip warning={true}>
+
+Our support for loading multiple LoRA checkpoints is currently limited and there might be rough edges involved.
```

Comment:
I would remove it here and move it below to the suggested part

Reply:
I think "limited support" and "rough edges" without specification may be a bit discouraging. I think it's best to maybe mention the known limitations in a more specific way.